@AH-64D Apache Longbow/Addons/fza_ah64_fire/functions/fn_damageEnginefire.sqf

```diff
@@ -0,0 +1,113 @@
+/* ----------------------------------------------------------------------------
+Function: fza_fire_fnc_damageEngineFire
+
+Description:
+    damageEngineFire
+
+Parameters:
+    _heli - The helicopter to modify
+    _eng - the engine that has been assigned to burn
+
+Returns:
+    Nothing
+
+Examples:
+    --- Code
+    [_heli, "right"] spawn fza_fire_fnc_damageEngineFire
+    ---
+
+Author:
+    Snow(dryden), Mattysmith22
+---------------------------------------------------------------------------- */
+params ["_heli","_eng"];
+
+_rand = floor random 3;
+if (_rand > 1) exitwith {};
+
+private _side = [];
+private _sidef = [];
+private _mag = "";
```

Comment:
If we are rebuilding this, maybe we should find a better name than `_mag` - `_fireId` or `_componentId`?